### ISSUE TITLE
What does model(images, targets) do?

### ISSUE THREAD

**Issue by DavidRydgard** [sagieppel/Train_Mask-RCNN-for-object-detection-in_In_60_Lines-of-Code#1]
Hi! Thank you very much for sharing this code, I might finally have found something that works.
However, I'm wondering what _model(images, targets)_ do. I understand that it returns a dictionary with the different losses, but I have never seen a call like this before, where you pass both inputs and targets to the model (I'm new to PyTorch). Where can I read about this? 
And how does it, in short, work, i.e. how can we get the loss without supplying a loss function ourselves? Are the different loss functions somehow stored in the model?
